gapic-generator/templates/default/service/client/method/def/_options_defaults.erb

```diff
@@ -1,6 +1,8 @@
 <%- assert_locals method -%>
 # Converts hash and nil to an options object
-options = Gapic::CallOptions.new options.to_h if options.respond_to? :to_h
+if options.respond_to? :to_h
+  options = Gapic::CallOptions.new **options.to_h
+end
```

Comment:
This also needs parens, and to be changed back to a modifier conditional. (Note that the rubocop line limit for these repos and the libraries they generate, is set to 120 rather than 80.)